### ISSUE TITLE
unmarshalling flake when fetching cluster status

### ISSUE THREAD

**Issue by Makdaam** [openshift/openshift-azure#1427]
```Control Plane Pods Status E2E tests [Fake][EveryPR]
/home/ljakubow/go/src/github.com/openshift/openshift-azure/test/e2e/specs/fakerp/clusterstatus.go:17
  should allow an SRE to fetch the status of control plane pods [It]
  /home/ljakubow/go/src/github.com/openshift/openshift-azure/test/e2e/specs/fakerp/clusterstatus.go:28

  Expected error:
      <*json.UnmarshalTypeError | 0xc0003bcaa0>: {
          Value: "array",
          Type: {
              size: 0x68,
              ptrdata: 0x58,
              hash: 3275050201,
              tflag: 7,
              align: 8,
              fieldAlign: 8,
              kind: 25,
              alg: {hash: nil, equal: nil},
              gcdata: 85,
              str: 78160,
              ptrToThis: 2813280,
          },
          Offset: 1,
          Struct: "",
          Field: "",
      }
      json: cannot unmarshal array into Go value of type v1.PodList
  not to have occurred
  /home/ljakubow/go/src/github.com/openshift/openshift-azure/test/e2e/specs/fakerp/clusterstatus.go:37
```
When fetching cluster status an array is returned instead of a PodList struct. Investigating.

/kind test-flake

**Comment by jim-minter** [comment]
Looks like (at least) two problems:
1) tests are not being focused in our e2es when they should be (this doesn't run every e2e but should - how many other tests are in that category? :(
2) test is probably just broken rather than being a flake - the back end is actually returning a `[]v1.Pod`

**Comment by jim-minter** [comment]
candidate tags to remove (probably among others): `[Admin]`, `[EveryPR]`

**Comment by jim-minter** [comment]
look at making a singleton AzureClient like, but *not in* sanity.Checker so we don't reinitialise it all the time

**Comment by Makdaam** [comment]
Fixed by #1437
